src/modules/admin/components/UserList/UserList.tsx

```diff
@@ -6,7 +6,7 @@ import { pipe, mergePayload, withKey } from '~utils/actions';
 import { useAsyncFunction } from '~utils/hooks';
 import { Table, TableBody } from '~core/Table';
 import Heading from '~core/Heading';
-import UserListItem from './UserListItem';
+import UserListItem from '../UserListItem/UserListItem';
 import styles from './UserList.css';
```

Comment:
I don't think this component will be needed anymore - it's only being used in the `Organizations` tab which I assume will be replaced by `Permissions`? For increased content flexibility, `UserListItem` is being used directly in `Permissions`.

Thus, I believe `UserList` can be removed via #1711